st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -325,8 +335,15 @@ def _run_action(self, action_node, parent_execution_id, params, wait_for_complet
         liveaction = LiveActionDB(action=action_node.ref)
         liveaction.parameters = action_param_utils.cast_params(action_ref=action_node.ref,
                                                                params=params)
-        if action_node.notify:
-            liveaction.notify = NotificationsHelper.to_model(action_node.notify)
+
+        # Setup notify for task in chain.
+        if action_node.name not in self._skip_notify_tasks:
```

Comment:
Might be a bit easier to read if it's refactored in a separate method - e.g. `_should_notify_for(task_name)`.